api/envoy/extensions/network/dns_resolver/cares/v3/cares_dns_resolver.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package envoy.extensions.network.dns_resolver.cares.v3;
+
+import "envoy/config/core/v3/address.proto";
+import "envoy/config/core/v3/resolver.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.network.dns_resolver.cares.v3";
+option java_outer_classname = "CaresDnsResolverProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: cares DNS resolver configuration]
+
+// Configuration for cares DNS resolver
```

Comment:
PTAL at the output of the docs build and see how this renders. I think you should use RST syntax below.

Reply:
ACK